met-web/src/services/userService/api/index.tsx

```diff
@@ -0,0 +1,31 @@
+import http from 'apiManager/httpRequestHandler';
+import Endpoints from 'apiManager/endpoints';
+import { Page } from 'services/type';
+import { User } from 'models/user';
+import { replaceUrl } from 'helper';
+
+interface GetUserParams {
+    page?: number;
+    size?: number;
+    sort_key?: string;
+    sort_order?: 'asc' | 'desc';
+}
+export const getUserList = async (params: GetUserParams = {}): Promise<Page<User>> => {
+    const responseData = await http.GetRequest<Page<User>>(Endpoints.User.GET_LIST, params);
+    return (
+        JSON.parse(JSON.stringify(responseData)).data ?? {
+            items: [],
+            total: 0,
+        }
+    );
```

Comment:
do you know why we have to do JSON.parse(JSON.stringify here..not a normal pattern..
Can you dig deeper ?

Reply:
hmmm, definitely looks weird. I'll check it.

Reply:
removed it